Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -34,15 +34,15 @@ class PDHG(Algorithm):
     Parameters
     ----------
     f : Function
-        A convex function with a "simple" proximal method of its conjugate.
+        A convex function with a "simple" proximal method of its conjugate. This function must map from the operator range to the Reals, as :math: `f(Kx)` or :math: `f^{*}(x)` will be the contribution to the total objective. See below for details.
```

Comment:
```suggestion
        A convex function with a "simple" proximal method of its conjugate. This function must map from the operator range to the Reals, as :math: `f(Kx)` will be the contribution to the total objective. See below for details.
```

I think leave the convex conjugate out for now?